### PR TITLE
feat: add job and script to delete blobs

### PR DESCRIPTION
**Pull request by tbpg**
The shape of this job might change in the future. But, to start, I'm keeping it as simple as possible. We can invoke the job multiple times to delete multiple tarballs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +100,16 @@ You can regenerate the HTML for a single blob by setting
...
 If you want to use a different bucket than the default, set `SOURCE_BUCKET`.
 
+### Deleting old content
+
+You can delete old tarballs using the `delete-blob` job. Trigger the job with
+the `BLOB_TO_DELETE` environment variable set to the full name of the blob
+you want to delete, for example `gs://my-bucket/lang-library-1.0.0.tar.gz`.
+
+Be sure to delete the `docfx-` and non-`docfx-` tarballs! Also, after deleting
+the tarball, be sure to delete the content on the site; it is not automatically
```
Comment (dandhlee) [inline README.md:110]: How do you delete content on the site? I'm not sure what's being referred by this

Comment (tbpg) [reply README.md:113]: Deleting from the bucket doesn't delete from cloud.google.com. Those pages still need to be deleted separately (internally).

Comment (dandhlee) [reply README.md:113]: I see! Thank you for clarifying that.

**Review by dandhlee — APPROVED** [review]
LGTM.

**Pull request merged**